lib/grinder_sdk.dart

```diff
@@ -17,6 +18,17 @@ import 'src/utils.dart';
 
 bool _sdkOnPath;
 
+// Returns the common top-level directories which usually contain Dart source
+// and which actually exist in the current project.
+Set<String> get defaultSourceDirectories => [
```

Comment:
I don't think `default` is necessary.  And we already have e.g. `binDir`, so we should similarly abbreviate `Directories` here.  Also, I think we wanted to have separate fields for existing vs. all, see discussion in #239. So I'd suggest:

```
Set<Directory> sourceDirs = ['bin', 'example', 'lib', 'test', 'tool', 'web'].map((path) => new Directory(path)).toSet();
Set<Directory> get existingSourceDirs => Directory.current.listSync().where(sourceDirs.contains).toSet();
```